### ISSUE TITLE
安装

### ISSUE THREAD

**Issue by Dittonal** [projectdone/DenseFuse-Refactoring-of-PyTorch#1]
- absl-py==1.4.0
- asttokens==2.2.1
- astunparse==1.6.3
- backcall==0.2.0
- backports.functools-lru-cache==1.6.4
- cachetools==4.2.4
- certifi==2022.12.7
- chardet==3.0.4
- charset-normalizer==3.0.1
- colorama==0.4.6
- contourpy==1.0.7
- cycler==0.11.0
- debugpy==1.5.1
- decorator==5.1.1
- distlib==0.3.6
- et-xmlfile==1.1.0
- executing==1.2.0
- filelock==3.9.0
- fonttools==4.38.0
- gast==0.3.3
- gluonts==0.12.2
- google-auth==1.35.0
- google-auth-oauthlib==0.4.6
- google-pasta==0.2.0
- graphviz==0.20.1
- grpcio==1.51.3
- h5py==2.10.0
- idna==2.6
- importlib-metadata==6.0.0
- importlib-resources==5.12.0
- ipykernel==6.15.0
- ipython==8.10.0
- jedi==0.18.2
- joblib==1.2.0
- jupyter_client==8.0.3
- jupyter_core==5.2.0
- Keras-Preprocessing==1.1.2
- kiwisolver==1.4.4
- Markdown==3.4.1
- MarkupSafe==2.1.2
- matplotlib==3.7.0
- matplotlib-inline==0.1.6
- mxnet==1.2.0
- nest-asyncio==1.5.6
- numpy==1.20.0
- oauthlib==3.2.2
- openpyxl==3.1.1
- opt-einsum==3.3.0
- orjson==3.8.6
- packaging==23.0
- pandas==1.2.0
- parso==0.8.3
- pickleshare==0.7.5
- Pillow==9.4.0
- platformdirs==3.0.0
- prompt-toolkit==3.0.36
- protobuf==3.20.0
- psutil==5.9.0
- pure-eval==0.2.2
- pyarrow==11.0.0
- pyasn1==0.4.8
- pyasn1-modules==0.2.8
- pydantic==1.10.5
- Pygments==2.14.0
- pyparsing==3.0.9
- python-dateutil==2.8.2
- pytz==2022.7.1
- pywin32==227
- pyzmq==23.2.0
- rdkit-pypi==2022.9.4
- requests==2.28.2
- requests-oauthlib==1.3.1
- rsa==4.9
- scikit-learn==1.0.2
- scipy==1.4.1
- six==1.16.0
- stack-data==0.6.2
- tensorboard==2.2.2
- tensorboard-plugin-wit==1.8.1
- tensorflow==2.2.0
- tensorflow-estimator==2.2.0
- termcolor==2.2.0
- threadpoolctl==3.1.0
- toolz==0.12.0
- torch==1.13.1+cpu
- torchaudio==0.13.1
- torchvision==0.14.1+cpu
- tornado==6.2
- tqdm==4.64.1
- traitlets==5.9.0
- typing_extensions==4.5.0
- urllib3==1.22
- virtualenv==20.19.0
- wcwidth==0.2.6
- Werkzeug==2.2.3
- wheel==0.38.4
- wincertstore==0.2
- wrapt==1.14.1
- zipp==3.14.0
